### PR TITLE
APILIB-20: Update python-api library to support query parameters for endpoints

### PR DESCRIPTION
**Pull request by gruberrro**
Possibly there should be a way to pass a query parameters to all requests.

### CODE REVIEW

File: infermedica_api/webservice.py [hunk 1]
```diff
@@ -182,11 +182,11 @@ def search(self, phrase, sex=None, max_results=8, filters=None, **kwargs):
         :rtype: list
         """
         method = self.__get_method('search')
-        params = {
+        params = kwargs.pop('params', kwargs)
```
Comment (gruberrro) [inline infermedica_api/webservice.py]: I'm not sure if it good approach...

File: infermedica_api/webservice.py [hunk 2]
```diff
@@ -238,14 +238,21 @@ def suggest(self, diagnosis_request, max_results=8, interview_id=None):
         """
         method = self.__get_method('suggest')
         headers = self.__get_interview_id_headers(interview_id=interview_id)
+        params = kwargs
```
Comment (gruberrro) [inline infermedica_api/webservice.py]: Either this or [this](https://github.com/infermedica/python-api/pull/8/files#r471714248)?

File: infermedica_api/webservice.py (hunk 1 above)
Comment (akszydelko) [reply infermedica_api/webservice.py]: Imo no, we don't know what will be put to kwargs. `params = kwargs.pop('params', {})` should be the way to go.

File: infermedica_api/webservice.py (hunk 2 above)
Comment (akszydelko) [reply infermedica_api/webservice.py]: Same here `params = kwargs.pop('params', {})`

**Review by akszydelko — CHANGES_REQUESTED** [review]

File: infermedica_api/webservice.py (hunk 1 above)
Comment (gruberrro) [reply infermedica_api/webservice.py]: @akszydelko done

File: infermedica_api/webservice.py (hunk 2 above)
Comment (gruberrro) [reply infermedica_api/webservice.py]: @akszydelko done

**Review by akszydelko — APPROVED** [review]

**Pull request merged**